README.md

```diff
@@ -5,13 +5,73 @@ This repository is for EDGI [Web Monitoring Project](https://github.com/edgi-gov
 
 EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently, there is a lot of manual labor that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
 
+- [Technologies Used](#hammer-technologies-used)
 - [Project Goals](#project-goals)
 - [How to Help](#how-to-help)
 - [Project Overview](#project-overview)
 - [License & Copyright](#license--copyright)
 
 You can **track upcoming releases** by exploring our [milestones](https://github.com/edgi-govdata-archiving/web-monitoring/milestones) within the GitHub issue queue.
 
+## :hammer: Technologies Used
+
+To help newcomers better understand where they might be able to
+contribute, these are the main tool and software that we use in this
+project. This includes software in the application, but also
+platforms that we rely on.
+
+And don't worry -- you definitely don't need to know all of them!
+
+* **HTML & CSS.**
+* **SASS.**
+* [**Ruby on Rails.**][rails] A web application development framework used by our backend API server.
+* **Javascript.** Notable packages include:
+    * [**Webpack.**][webpack] A _static module bundler_ for modern JavaScript
+      applications.
+    * [**ReactJS.**][reactjs] A JavaScript library for building user
+      interfaces.
+* **Python.** Notable packages include:
+    * [**Beautiful Soup.**][beautifulsoup] A library for pulling data out
+      of HTML and XML files.
+    * [**Tornado.**][tornado] A web server and application framework.
+* [**Postgresql.**][postgres] A powerful, open source
+  object-relational database system.
+* [**Redis.**][redis] An open source, in-memory data structure store,
+  used as a database, cache and message broker.
+* [**Swagger.**][swagger] A spec and framework for API developer tools.
+* [**Heroku.**][heroku] A platform for easily deploying applications.
+* [**Review Apps.**][review-apps] A Heroku feature that deploys code
+  from GitHub pull requests as a disposable app on the web.
+* [**Ansible.**][ansible] An open source automation platform for
```

Comment:
Feels weird to say this if we haven’t actually decided to use it yet.

Reply:
I was feeling some pretty strong vibes toward it, but maybe I misread. This list is mainly about letting people know that their experience and knowledge is welcome, so if it's something we're gunning for using (and would appreciate people with expertise), then I'd love to flag that prominently.

What if we agreed to remove if/when we no longer have interest in that expertise?

cc @danielballan